azkaban-exec-server/src/main/java/azkaban/execapp/FlowRunnerManager.java

```diff
@@ -732,6 +733,19 @@ public void shutdownNow() {
     logger.warn("Shutting down FlowRunnerManager now...");
     this.executorService.shutdownNow();
     this.triggerManager.shutdown();
+    deleteExecutionDirectory();
```

Comment:
@HappyRay it's simpler, you can compare the old commit with the latest one with suggested change.